src/memory-node/memory_server_main.cpp

```diff
@@ -89,9 +89,15 @@ int main(int argc, char *argv[]) {
                 << "\n"
                 << "\t-v/--version        : Display metadata server version  \n"
                 << "\n"
+				<< "\t-i/--init			  : Initialize the root shelf \n"
+				<< "\n"
                 << endl;
             exit(0);
-        } else if ((std::string(argv[i]) == "-a") ||
+		} else if ((std::string(argv[i]) == "-i") ||
+                   (std::string(argv[i]) == "--init")) {
+            StartNVMM();
```

Comment:
StartNVMM() needs to take fam path. May be you can use -f parameter also together with this.